ci-operator/step-registry/acm/fetch-managed-clusters/README.md

```diff
@@ -0,0 +1,31 @@
+# acm-fetch-managed-clusters-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+
+## Purpose
+
+To gather information about the managed clusters that are created as a result of the clc-test-create-ref.
+
+## Process
+
+- Set two dynamic variables based on the ephemeral hub cluster that needs to be provisioned prior to running this.
+- Runs a [script from product QE's repo](https://github.com/stolostron/acmqe-autotest/blob/main/ci/containerimages/fetch-managed-clusters/fetch_clusters_commands.sh) that does some additional config and ultimately runs a python script ([generate_managedclusters_data.py](https://github.com/stolostron/acmqe-autotest/blob/main/ci/containerimages/fetch-managed-clusters/generate_managedclusters_data.py)) to fetch information about the managed clusters.
+
+## Requirements
+
+
+### Infrastructure
+
+- A provisioned test cluster to target (hub).
+- "advanced-cluster-management" operator installed (see [`install-operators`](../../../step-registry/install-operators/README.md)).
```

Comment:
Does the operator need to be installed in a specific namespace or all of the namespaces? Not sure if that affects this script

Reply:
I havn't tested if this can be installed into a global namespace. This won't effect this script though. 

I wouldn't expect anyone to make use of this without having a managed cluster installed using ACM, meaning the operator would've been needed in a prior step along with other things. If they did and had ACM operator installed in some other namespace specific or global I doubt this will matter, I couldn't find any reference so a namespace anywhere in the scripts.